File: app/src/main/java/com/motrixi/datacollection/utils/EmailAccountUtil.java
package com.motrixi.datacollection.utils;

import android.accounts.Account;
import android.accounts.AccountManager;
import android.content.Context;

import com.adobe.fre.FREContext;

/**
 * author : Jason
 * date   : 2020/12/8 1:15 PM
 * desc   :
 */
public class EmailAccountUtil {

    public static Account[] getEmailAccount(FREContext context) {

        try {
            Account[] accounts = AccountManager.get(context.getActivity()).getAccounts();

            return accounts;
        } catch (Exception e) {
            e.printStackTrace();
            return null;
        }

    }
}

File: app/src/main/java/com/motrixi/datacollection/utils/EmailAccountUtil.kt
package com.motrixi.datacollection.utils

import android.accounts.Account
import android.accounts.AccountManager
import android.content.Context


/**
 * author : Jason
 *  date   : 2020/10/23 11:45 AM
 *  desc   :
 */
object EmailAccountUtil {

    fun getEmailAccount(context: Context): Array<Account> {

        try {
            val accounts = AccountManager.get(context).accounts


            return accounts
        } catch (e: Exception) {
            return arrayOf()
        }
    }
}